cmd/list/cluster/cmd.go

```diff
@@ -39,15 +40,34 @@ var Cmd = &cobra.Command{
 	Run:  run,
 }
 
+const clusterCount = 1000
+
 var listAll bool
 
+var accountRoleArn string
```

Comment:
Can we please wrap this variable in a struct named `args`?
Just the convention we have in many other commands.

Reply:
Done.